tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -575,7 +606,7 @@ def test_seer_call(self, mock_seer_request: MagicMock):
         self.assert_actions_resolved_for_incident(
             incident,
             [warning_action],
-            [(warning_action, incident, self.project, 1, IncidentStatus.CLOSED, mock.ANY)],
+            [(warning_action, incident, self.project, IncidentStatus.CLOSED, 1, mock.ANY)],
```

Comment:
Converting these to use kwargs/dictionaries would probably be a lot safer than the kludge to map these tuples to kwargs based on position. Though this will be a bit more work, could we update this to be less fragile please?